.gitmodules

```diff
@@ -38,3 +38,6 @@
 [submodule "3rdparty/openssl/intel-sgx-ssl-3"]
 	path = 3rdparty/openssl/intel-sgx-ssl-3
 	url = https://github.com/intel/intel-sgx-ssl.git
+[submodule "3rdparty/symcrypt_provider/SymCrypt-OpenSSL"]
```

Comment:
we already pull this repo under symcrypt_engine. Can't we use the same branch for both engine and provider?

Reply:
I haven't tested if the newer versions of SymCrypt-OpenSSL would be backward compatible with the SymCrypt engine used for symcrypt_engine

Reply:
Given that the engine and provider implementation is separated in the repository, we can give it a try to see if we can just have one clone?

Reply:
@mingweishih There might be changes in a later commit of SCOSSL that might impact SymCrypt engine scenario, do you have any concerns about that? We might be able to move some tests in openssl_symcrypt.unsupported to supported

Reply:
let's investigate this in a separate PR